src/sql/SqlPage.ts

```diff
@@ -53,6 +54,7 @@ export class SqlPage {
 
     /**
      * This is needed for better mocking. Constructor mocking is not trivial with sinon.
+     * @internal
```

Comment:
Do we need this, since the class is marked as internal?

Reply:
👍🏻  no we don't